mmdet/models/necks/nas_fpn.py

```diff
@@ -123,24 +65,53 @@ def __init__(self,
 
         # add NAS FPN connections
         self.fpn_stages = nn.ModuleList()
+        out_conv_cfg = dict(groups=1, kernel_size=3, padding=1, bias=True)
```

Comment:
`groups=1` and `bias=True` can be omitted.

Reply:
I think they cannot be omitted because I use `out_conv_cfg['groups']` to get the value so that the dict is expected to contain key `groups` (so as `bias` ).

In the latest commit, I use `out_conv_cfg` in nas_fpn as the default argument while building 
`BaseMergeCell`

Reply:
`**out_conv_cfg` could help.